src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="search search--big"
+      data-qa="big"
+    >
+      <div class="search__icon search__icon--big"></div>
+      <input
+        class="search__input search__input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search search--small"
+      data-qa="small"
+    >
+      <div class="search__icon search__icon--small"></div>
+      <input
+        class="search__input search__input--small"
+        type="text"
+        data-qa="keypress"
```

Comment:
**add attribute data-qa="keypress" to input in big search form**

remove this for search__input--small